app/commands/user/reputation_token/award_for_issue.rb

```diff
@@ -0,0 +1,47 @@
+class User
+  class ReputationToken
+    class AwardForIssue
+      include Mandate
+
+      initialize_with :params
+
+      def call
+        return unless label_change?
+        return unless reputation_level
+        return unless author
+        return unless author.maintainer? || author.admin?
```

Comment:
I could change this to:

```suggestion
        return unless author&.maintainer? || author&.admin?
```

but I was unsure whether this is idiomatic

Reply:
Yeah - this is fine.